lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -56,6 +56,12 @@ def self.available_options
                                        optional: true,
                                        default_value: '%B',
                                        is_string: true),
+          FastlaneCore::ConfigItem.new(key: :match_pattern,
+                                       env_name: 'FL_CHANGELOG_FROM_GIT_COMMITS_MATCH_PATTERN',
+                                       description: 'Whether or not to match a glob(7) pattern in tag name',
+                                       optional: true,
+                                       default_value: nil,
+                                       is_string: false),
```

Comment:
This option is a `String`, yes? If so, `is_string` will default to `true` so this can be omitted.